docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.rst

```diff
@@ -1,18 +1,110 @@
+
+2. **Identify your Data Context Expectations Store**
+
+    In your ``great_expectations.yml`` , look for the following lines.  The configuration tells Great Expectations to look for Expectations in a store called ``expectations_store``. The ``base_directory`` for ``expectations_store`` is set to ``expectations/`` by default.
+
+
+    .. code-block:: yaml
+
+        expectations_store_name: expectations_store
+
+        stores:
+            expectations_store:
+                class_name: ExpectationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: expectations/
+
+
+3. **Update your configuration file to include a new store for Expectations on GCP.**
+
+    In our case, the name is set to ``expectations_GCP_store``, but it can be any name you like.  We also need to make some changes to the ``store_backend`` settings.  The ``class_name`` will be set to ``TupleGCSStoreBackend``, ``project`` will be set to the GCP project, ``bucket`` will be set to the address of your GCP bucket, and ``prefix`` will be set to the folder where Expectation files will be located.
+
+
+.. warning::
+
+    If you are also storing :ref:`Validations in GCP, <how_to_guides__configuring_metadata_stores__how_to_configure_a_validation_result_store_in_gcs>` please ensure that the ``prefix`` values are disjoint and one is not a substring of the other.
+
+    .. code-block:: yaml
+
+        expectations_store_name: expectations_GCP_store
+        stores:
+            expectations_GCP_store:
+                class_name: ExpectationsStore
+                store_backend:
+                    class_name: TupleGCSStoreBackend
+                    project: '<your_GCP_project_name>'
+                    bucket: '<your_GCP_bucket_name>'
+                    prefix: '<your_GCP_folder_name>'
+
+
+4. **Copy existing Expectation JSON files to the GCP bucket**. (This step is optional).
+
+    One way to copy Expectations into GCP is by using the `gsutil cp` command, which is part of the Google Cloud SDK. The command will copy one Expectation, ``exp1`` from a local folder to the GCP bucket.   Information on other options, like the Cloud Storage browser in the Google Cloud Console, can be found in the `Documentation for Google Cloud <https://cloud.google.com/storage/docs/uploading-objects>`_.
+
+    .. code-block:: bash
+
+        gsutil cp exp1.json gs://'<your_GCP_bucket_name>'/'<your_GCP_folder_name>'
+
+        Operation completed over 1 objects/58.8 KiB.
+
+
+
+5. **Confirm that the new Expectations store has been added by running** ``great_expectations store list``.
+
+    Notice the output contains two Expectation stores: the original ``expectations_store`` on the local filesystem and the ``expectations_GCP_store`` we just configured.  This is ok, since Great Expectations will look for Expectations in GCP as long as we set the ``expectations_name`` variable to ``expectations_GCP_store``.
```

Comment:
Should we tell users that it's okay to delete the config for the older store?

Reply:
added this :)